.github/workflows/docker-publish.yml

```
@@ -120,9 +120,9 @@ jobs:
           environment: prod
           summary: true
 
-      - name: Upload SARIF result
-        id: upload-sarif
-        if: ${{ github.event_name != 'pull_request_target' }}
-        uses: github/codeql-action/upload-sarif@38697555549f1db7851b81482ff19f1fa5c4fedc # v4.34.1
-        with:
-          sarif_file: sarif.output.json
+      #- name: Upload SARIF result
```

YAMLlint (1.38.0)

[warning] missing starting space in comment

(comments)